web/packages/teleport/src/services/notifications/types.ts

```diff
@@ -85,8 +85,6 @@ export enum NotificationSubKind {
   AccessRequestPending = 'access-request-pending',
   AccessRequestApproved = 'access-request-approved',
   AccessRequestDenied = 'access-request-denied',
-  /** AccessRequestNowAssumable is the notification for when an approved access request that was scheduled for a later date is now assumable. */
```

Comment:
This type of notification isn't feasible because we don't support creating a notification at some point in the future.